### PR TITLE
Addressing remaining IESG comments

### PR DESCRIPTION
**Pull request by linuxwolf**
This PR addresses the remaining comments from Ben Kaduk.

### CODE REVIEW

**Comment by linuxwolf** [comment]
@kaduk do these changes match your expectations?

File: draft-ietf-cose-rfc8152bis-struct.xml [hunk 1]
```diff
@@ -1658,14 +1658,16 @@ encryptedKey = KeyWrap(KDF(DH-Shared, context), CEK)
       <name>IANA Considerations</name>
       <t>
         The registries and registrations listed below were created during processing of RFC 8152 <xref target="RFC8152"/>.
-        The majority of the actions are to update the references to point to this document.
+        The majority of the following actions are to update the references to point to either this document.
```
Comment (kaduk) [inline draft-ietf-cose-rfc8152bis-struct.xml]:
```suggestion
        The majority of the following actions are to update the references to point to this document.
```

File: draft-ietf-cose-rfc8152bis-struct.xml [hunk 2]
```diff
@@ -1658,14 +1658,16 @@ encryptedKey = KeyWrap(KDF(DH-Shared, context), CEK)
...
       </t>
       <section anchor="cose-header-key-table">
         <name>COSE Header Parameters Registry</name>
         <t>
           IANA created a registry titled "COSE Header Parameters" as part of processing <xref target="RFC8152"/>. 
-          IANA is requested to update the reference for all entries, except "counter signature" and "CounterSignature0", in the table from <xref target="RFC8152"/> to this document.
-          The reference for "counter signature" and "CounterSignature0" are to be left alone.
+          IANA is requested to update the reference for this registry from <xref targe="RFC"/> to this document.  IANA is also requested to update the reference for all entries, except "counter signature" and "CounterSignature0", in the table from <xref target="RFC8152"/> to this document.  The reference for "counter signature" and "CounterSignature0" are to be left as-is.
```
Comment (kaduk) [inline draft-ietf-cose-rfc8152bis-struct.xml]:
```suggestion
          IANA is requested to update the reference for this registry from <xref target="RFC8152"/> to this document.  IANA is also requested to update the reference for all entries, except "counter signature" and "CounterSignature0", in the table from <xref target="RFC8152"/> to this document.  The reference for "counter signature" and "CounterSignature0" are to be left as-is.
```

File: draft-ietf-cose-rfc8152bis-struct.xml [hunk 3]
```diff
@@ -1674,7 +1676,7 @@ encryptedKey = KeyWrap(KDF(DH-Shared, context), CEK)
         <name>COSE Key Common Parameters Registry</name>
         <t>
           IANA created a registry titled "COSE Key Common Parameters" as part of the processing of <xref target="RFC8152"/>.
-          IANA is requested to update the reference for entries in the table from <xref target="RFC8152"/> to this document.
+          IANA is requested to update the registry from <xref target="RFC8152"/> to this document.  IANA is also requested to update the reference for entries in the table from <xref target="RFC8152"/> to this document.
```
Comment (kaduk) [inline draft-ietf-cose-rfc8152bis-struct.xml]:
```suggestion
          IANA is requested to update the reference for this registry from <xref target="RFC8152"/> to this document.  IANA is also requested to update the reference for entries in the table from <xref target="RFC8152"/> to this document.
```

**Review by kaduk — COMMENTED** [review]
This looks pretty good (some suggestions inline).

Comparing against my ballot position (at https://datatracker.ietf.org/doc/draft-ietf-cose-rfc8152bis-struct/ballot/), I don't see anything to match:

> It seems that https://www.rfc-editor.org/errata/eid5066 needs to be
incorporated still (and the report verified, presuming it is correct).

but my recollection is that that one is already incorporated, just into -algs, and I should mark it verified

and

> We lost the section about the "CBOR Tags" registry (where we were previously
allocating a codepoint for COSE_Signature).  I think we need to bring back a
stub section with the directive to update the references from 8152 to this
document.

which may still be outstanding (I don't remember any other clarifications about that point having gone by).

**Comment by linuxwolf** [comment]
> This looks pretty good (some suggestions inline).
> 
> Comparing against my ballot position (at https://datatracker.ietf.org/doc/draft-ietf-cose-rfc8152bis-struct/ballot/), I don't see anything to match:
> 
> > It seems that https://www.rfc-editor.org/errata/eid5066 needs to be
> > incorporated still (and the report verified, presuming it is correct).
> 
> but my recollection is that that one is already incorporated, just into -algs, and I should mark it verified
> 

Correct; it is in § 9 of -algs, which is also the only document the KDF_Context is defined (or mentioned).
> and
> 
> > We lost the section about the "CBOR Tags" registry (where we were previously
> > allocating a codepoint for COSE_Signature).  I think we need to bring back a
> > stub section with the directive to update the references from 8152 to this
> > document.
> 
> which may still be outstanding (I don't remember any other clarifications about that point having gone by).

Ah, that's the one.  Will add.

**Comment by linuxwolf** [comment]
I do note that Göran's name is still too "ASCII-ized"; how to include the unicode here is not immediately clear (it's not in an \<author/>, it's not \<artwork>, etc).  I suggest adding an RFC Editor's note about it?

**Comment by linuxwolf** [comment]
@kaduk  Looking again, there is a CBOR Tags section in this document, at the end of the IANA section instead of the beginning.  I think we're good to go, once this is merged.

**Pull request merged**